unstoppable-ios-app/domains-manager-ios/Services/Firebase/Purchase/FirebasePurchaseDomainsService.swift

```diff
@@ -80,6 +81,17 @@ extension FirebasePurchaseDomainsService: PurchaseDomainsServiceProtocol {
         return domains
     }
     
+    func getDomainsSuggestions(hint: String?) async throws -> [DomainToPurchaseSuggestion] {
+        let hint = hint ?? "Anything you think is trending now"
+        let queryComponents = ["phrase" : hint,
+                               "extension" : "All"]
+        let urlString = URLSList.DOMAIN_AI_SUGGESTIONS_URL.appendingURLQueryComponents(queryComponents)
```

Comment:
would be nice to extract the URL creation to am appropriate file like NetworkService